app/authentication_views.py

```diff
@@ -17,27 +22,28 @@ def register():
         username = request.form['username']
         email = request.form['email']
         password = request.form['password']
-        birthDay = request.form['birthDay'].split("-")[0]
+        birthday = request.form['birthday']
 
+        # TODO change return statements
+        if not validation.validate_date_format(birthday):
+            return "wrong date format"
 
-        user = User(username, email, password, birthDay)
-
-        # TODO change responses later
-        if user.ifUserExists():
+        if UsersRepository.getByEmailUsername(username, email):
             return "User already exits"
-        if not user.validatePassword():
+
+        if not validation.validate_password(password):
             return "at least 1 capital letter, 1 small letter, 1 number, length 8 - 20"
 
-        if not user.validateEmail():
+        if not validation.validate_email(email):
             return "email has wrong format"
```

Comment:
few lines before that you have 'wrong date format' so keep same convention and go for 'wrong email format'

Reply:
Todo for next PR, extract all error messages to separate file

Reply:
this way you can see them next to each other and easily spot inconsistencies like this and reuse some generic error messages like 'something went wrong'

Reply:
added TODO